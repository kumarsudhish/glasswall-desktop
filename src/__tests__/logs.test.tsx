import React from 'react';
import { shallow } from 'enzyme'
import Header from '../ui/components/Header';
import { configure } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

configure({ adapter: new Adapter() })
// describe(name, fn) creates a block that groups together several related tests in one "test suite".
describe('`Header Snapshots', () => {
    it('should render our Snapshots correctly', () => {
      const wrapper = shallow(<Header showBack={false} />)
      expect(wrapper).toMatchSnapshot()
    })
  })