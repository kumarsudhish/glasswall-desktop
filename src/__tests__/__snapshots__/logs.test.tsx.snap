// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`\`Header Snapshots should render our Snapshots correctly 1`] = `
<div
  className="makeStyles-header-4"
>
  <div
    className="makeStyles-logoSection-5"
  >
    <img
      className="makeStyles-logo-6"
      src={null}
    />
  </div>
  <Navbar />
</div>
`;
